### PR TITLE
Avan1235/fix master workflows

### PR DESCRIPTION
**Pull request by avan1235**
fixes api naming to 'cafeterias' in urls
fix indentation in android worflow build

### CODE REVIEW

File: django-server/server/api/urls.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 from . import views
 
 router = routers.DefaultRouter()
-router.register(r'cafeteria', views.CafeteriaViewSet)
+router.register(r'cafeterias', views.CafeteriaViewSet)
```
Comment (kantoniak) [inline django-server/server/api/urls.py:6]: Code added by accident? Please move to a separate PR.

File: .github/workflows/android_apk_build_dev.yaml [hunk 2]
```diff
@@ -29,15 +28,16 @@ jobs:
...
+          wget https://eaterslab-dev.herokuapp.com/api.yaml                                       # get current api specification from server
           chmod +x ./api_generate_kotlin.sh
           ./api_generate_kotlin.sh api.yaml
           export GRADLE_USER_HOME=$(pwd)/.gradle
           chmod +x ./gradlew
           ./gradlew assembleDebug                                                             # run gradle compilation process in android project directory
-          curl -H 'Authorization: Token ${{ secrets.TOKEN_ADMIN_DEV }}' -X PUT --data-binary @./app/build/outputs/apk/debug/app-debug.apk https://eaterslab-dev.herokuapp.com/api/beta/upload/artifacts/beta/file/ # send file to server
+          curl -H 'Authorization: Token ${{ secrets.TOKEN_ADMIN_DEV }}' -X PUT --data-binary @./app/build/outputs/apk/debug/app-debug.apk https://eaterslab-dev.herokuapp.com/api/beta/upload/artifacts/beta/file/ # send file to server dev
+          curl -H 'Authorization: Token ${{ secrets.TOKEN_ADMIN }}' -X PUT --data-binary @./app/build/outputs/apk/debug/app-debug.apk https://eaterslab.herokuapp.com/api/beta/upload/artifacts/beta/file/ # send file to server
```
Comment (kantoniak) [inline .github/workflows/android_apk_build_dev.yaml:38]: API is fetched from dev and then APK goes to main (release?) server. Is this expected? I would expect this APK is not interoperable with different API versions.

Comment (kantoniak) [reply .github/workflows/android_apk_build_dev.yaml:38]: If app works with any two versions, can we add a comment here?

File: django-server/server/api/urls.py (hunk 1 above)
Comment (avan1235) [reply django-server/server/api/urls.py:6]: done in #89

File: .github/workflows/android_apk_build_dev.yaml (hunk 2 above)
Comment (avan1235) [reply .github/workflows/android_apk_build_dev.yaml]: I wanted to have two version of apps on every server but thanks to you I realized it's not a good way to go. Remove debug apk from release server and release apk from dev server
